message_subscribe_email/message_subscribe_email.module

```diff
@@ -55,6 +55,14 @@ function message_subscribe_email_message_subscribe_get_subscribers_alter(array &
     $flag_ids[] = $flag->id();
   }
 
+  // If email is one of the default notifiers, it should be unset prior to
```

Comment:
This is looking a bit weird. I would expect the default notifier to add itself, rather than unsetting it here.